# HG changeset patch
# User Jacob Dawid <jacob.dawid@googlemail.com>
# Date 1311700446 -7200
#      Tue Jul 26 19:14:06 2011 +0200
# Node ID ac64163efbe8b4410d86596336a13dea05d94573
# Parent  a43ecce77eec3ba36f999bd2f9cd4606deccf2dc
Deactivated buttons that are without function.

diff --git a/gui/src/SettingsDialog.ui b/gui/src/SettingsDialog.ui
--- a/gui/src/SettingsDialog.ui
+++ b/gui/src/SettingsDialog.ui
@@ -211,16 +211,19 @@
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="resetButton">
+       <property name="enabled">
+        <bool>false</bool>
+       </property>
        <property name="text">
         <string>Reset to defaults</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
@@ -231,23 +234,29 @@
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="exportButton">
+       <property name="enabled">
+        <bool>false</bool>
+       </property>
        <property name="text">
         <string>Export</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="importButton">
+       <property name="enabled">
+        <bool>false</bool>
+       </property>
        <property name="text">
         <string>Import</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
